Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -637,6 +648,15 @@ extension ActivityStreamPanel: DataObserverDelegate {
     }
 }
 
+extension ActivityStreamPanel: UIPopoverPresentationControllerDelegate {
+
+    // Dismiss the popover if the device is being rotated.
+    // This is used by the Share UIActivityViewController action sheet on iPad
+    func popoverPresentationController(_ popoverPresentationController: UIPopoverPresentationController, willRepositionPopoverTo rect: UnsafeMutablePointer<CGRect>, in view: AutoreleasingUnsafeMutablePointer<UIView>) {
```

Comment:
Are there any other reasons this delegate method might get called besides device rotation that we should be concerned about?